### PR TITLE
fix icon for failed test

### PR DESCRIPTION
**Pull request by antipeon**

### CODE REVIEW

**Comment by antipeon** [comment]
Squash + check tests again for [pr-188](https://github.com/UnitTestBot/UTBotCpp/pull/188)

**Comment by ladisgin** [comment]
check tests again

File: server/test/framework/Server_Tests.cpp [hunk 1]
```diff
@@ -1486,7 +1487,11 @@ namespace {
 
         ASSERT_FALSE(statusMap.empty());
         EXPECT_GT(statusMap.getNumberOfTests(), 2);
-        testUtils::checkStatuses(statusMap, tests);
+
+        StatusCountMap expectedStatusCountMap{
+            {testsgen::TEST_FAILED, 4},
```
Comment (ladisgin) [inline server/test/framework/Server_Tests.cpp:1492]: We wait that our tests failed?

File: server/test/framework/Syntax_Tests.cpp [hunk 2]
```diff
@@ -2062,16 +2062,14 @@ namespace {
...
+        utbot::SettingsContext settingsContext{ true, false, 45, 0, false, false };
         coverageGenerator.generate(false, settingsContext);
 
         EXPECT_FALSE(coverageGenerator.hasExceptions());
         ASSERT_TRUE(coverageGenerator.getCoverageMap().empty());
 
         auto statusMap = coverageGenerator.getTestStatusMap();
         auto tests = coverageGenerator.getTestsToLaunch();
-
-        testUtils::checkStatuses(statusMap, tests);
```
Comment (ladisgin) [inline server/test/framework/Syntax_Tests.cpp:2074]: Looks like this test doesn’t check anything

**Review by ladisgin — CHANGES_REQUESTED** [review]

File: server/test/framework/TestUtils.cpp [hunk 3]
```diff
@@ -132,11 +132,15 @@ namespace testUtils {
     }
 
     void checkStatuses(const Coverage::TestStatusMap &testStatusMap,
-                       const vector<UnitTest> &tests) {
+                       const vector<UnitTest> &tests,
+                       const StatusCountMap &expectedStatusCountMap) {
+        StatusCountMap actualStatusCountMap;
         for (auto const &[filename, suitename, testname] : tests) {
             const auto status = testStatusMap.at(filename).at(testname);
-            EXPECT_TRUE((testsgen::TestStatus::TEST_PASSED == status) ||
```
Comment (ladisgin) [inline server/test/framework/TestUtils.cpp:138]: We don’t need delete check

**Comment by antipeon** [comment]
Hanging until failed tests are removed

**Comment by antipeon** [comment]
remove status check for error suite

**Review by ladisgin — APPROVED** [review]

**Pull request merged**